buildSrc/src/main/groovy/alkemy.publish-conventions.gradle

```diff
@@ -0,0 +1,40 @@
+publishing {
+  repositories {
+    mavenLocal()
+    maven {
+      url = 'https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/'
+      credentials {
+        username = System.getenv('NEXUS_USERNAME')
+        password = System.getenv('NEXUS_PASSWORD')
```

Comment:
I changed it to use `System#getenv(String)` so that Gradle will not fail to load the project if these environment variables are not defined (e.g. for contributors like me who do not have the credentials to open the project in their local environment)

Reply:
Good call!